### ISSUE TITLE
Speed up paneViewSlideOffAnimation

### ISSUE THREAD

**Issue by jeanelie** [erichoracek/MSDynamicsDrawerViewController#33]
It would be nice if it was possible to set the paneViewSlideOff time to speed it up.

**Comment by iltercengiz** [comment]
:+1:  same here.

**Comment by mureev** [comment]
+1

**Comment by mureev** [comment]
I am able to fix this by overriding one method and replace first part of this animation by regular animationWithDuration. I don't think that it's good solution but I could share it with others. And it's totally solve this issue!

**Comment by CameronBanga** [comment]
@mureev could you share that? Looking to speed up as well!

**Comment by mureev** [comment]
- Subclass MSDynamicsDrawerViewController
- Add extension in your subclass

``` objective-c
@interface MSDynamicsDrawerViewController ()

@property (nonatomic, strong) UIDynamicAnimator *dynamicAnimator;

- (CGPoint)paneViewOriginForPaneState:(MSDynamicsDrawerPaneState)paneState;

@end
```
- Override setPaneState: inDirection: animated:

``` objective-c
- (void)setPaneState:(MSDynamicsDrawerPaneState)paneState inDirection:(MSDynamicsDrawerDirection)direction animated:(BOOL)animated allowUserInterruption:(BOOL)allowUserInterruption completion:(void (^)(void))completion {
    if (paneState == MSDynamicsDrawerPaneStateOpenWide) {
        [self.dynamicAnimator removeAllBehaviors];
        [UIView animateWithDuration:0.2 animations:^{
            self.paneView.frame = (CGRect){[self paneViewOriginForPaneState:paneState], self.paneView.frame.size};
        } completion:^(BOOL finished) {
            if (completion) {
                completion();
            }
        }];
    } else {
        [super setPaneState:paneState inDirection:direction animated:animated allowUserInterruption:allowUserInterruption completion:completion];
    }
}
```

**Comment by erichoracek** [comment]
I've added a fix for this in #55. Check it out and let me know if you have any questions or comments. It should _by default_ be a faster transition with the same effect.
